Update 01-match-clause.md

zaWandererer opened the pull request:
changed `has` to `owns`.  and the corresponding TypeQL code for this query.

zaWandererer committed:
Update 01-match-clause.md

changed `has` to `owns`.  and the corresponding TypeQL code for this query

alexjpwalker commented:
Thanks for the contribution, @zaWandererer !

Let's also fix the Java code (just a few lines below this change):

```java
TypeQLMatch.Filtered query = TypeQL.match(
  var("x").owns("title")
).get("x");
```

zaWandererer committed:
Update 01-match-clause.md

modified the java code per instructions in https://github.com/vaticle/docs/pull/641#issuecomment-1354626430
